docs/src/components/HomepageFeatures/index.tsx

```diff
@@ -11,7 +11,7 @@ type FeatureItem = {
 const FeatureList: FeatureItem[] = [
   {
     title: 'Pure Java',
-    Svg: require('@site/static/img/pure-java.svg').default,
```

Comment:
I don't feel strongly, but the original with just a `0` looks better to me than the `0%` in the new one.

Reply:
done :+1: